backend/routes/sequence.js

```diff
@@ -156,12 +197,88 @@ router.get("/revision/:revId/sequences/:sequenceId/activities/:activityId", midd
  */
 router.get("/revision/master/head/sequences/:sequenceId/activities", middlewares.issue.canView, getSequenceActivities);
 router.get("/revision/:revId/sequences/:sequenceId/activities", middlewares.issue.canView, getSequenceActivities);
+/**
+ * @api {get} /:teamspace/:model/sequences/:sequenceId/activities Get all activities
+ * @apiName getSequenceActivities
+ * @apiGroup Sequences
+ * @apiDescription Get all sequence activities.
+ *
+ * @apiUse Sequences
+ *
+ * @apiParam {String} sequenceId Sequence unique ID
+ *
+ * @apiExample {get} Example usage
+ * GET /acme/00000000-0000-0000-0000-000000000000/sequences/00000000-0000-0000-0001-000000000001/activities HTTP/1.1
+ *
+ * @apiSuccessExample {json} Success-Response
+ * HTTP/1.1 200 OK
+ * {
+ * 	"tasks":[
+ * 		{
+ * 			"id":"00000000-0000-0001-0001-000000000001",
+ * 			"name":"Construction",
+ * 			"startDate":1244246400000,
+ * 			"endDate":1244246450000,
+ * 			"subTasks":[
+ * 				{
+ * 					"id":"00000000-0001-0001-0001-000000000001",
+ * 					"name":"Prepare site",
+ * 					"startDate":1244246400000,
+ * 					"endDate":1244246430000,
+ * 					"subTasks":[
+ * 						{
+ * 							"id":"00000001-0001-0001-0001-000000000001",
+ * 							"name":"Erect site hoarding",
+ * 							"startDate":1244246400000,
+ * 							"endDate":1244246410000
+ * 						},
+ * 						{
+ * 							"id":"00000002-0001-0001-0001-000000000001",
+ * 							"name":"Clear existing structures",
+ * 							"startDate":1244246410000,
+ * 							"endDate":1244246420000
+ * 						},
+ * 						{
+ * 							"id":"00000003-0001-0001-0001-000000000001",
+ * 							"name":"Smooth work surfaces",
+ * 							"startDate":1244246420000,
+ * 							"endDate":1244246430000
+ * 						}
+ * 					]
+ * 				},
+ * 				{
+ * 					"id":"00000001-0002-0001-0001-000000000001",
+ * 					"name":"Construct tunnel",
+ * 					"startDate":1244246430000,
+ * 					"endDate":1244246450000,
+ * 					"subTasks":[
+ * 						{
+ * 							"id":"00000001-0002-0001-0001-000000000001",
+ * 							"name":"Deploy instant tunnel",
+ * 							"startDate":1244246430000,
+ * 							"endDate":1244246440000
+ * 						},
+ * 						{
+ * 							"id":"00000002-0002-0001-0001-000000000001",
+ * 							"name":"Add road markings",
+ * 							"startDate":1244246440000,
+ * 							"endDate":1244246450000
+ * 						}
+ * 					]
+ * 				}
+ * 			]
+ * 		}
+ * 	]
+ * }
+ */
+router.get("/sequences/:sequenceId/activities", middlewares.issue.canView, getSequenceActivities);
 
 /**
  * @api {get} /:teamspace/:model/revision(/master/head/|/:revId)/sequences/:sequenceId/state/:stateId Get state
- * @apiName getSequenceState
+ * @apiName getSequenceStateOld
```

Comment:
just ditch these APIs, we're not providing backwards support since we don't think there's any users who're using the APIs for this atm.